path_role.go

```diff
@@ -70,6 +73,24 @@ should never expire. The token should be renewed within the
 duration specified by this value. At each renewal, the token's
 TTL will be set to the value of this parameter.`,
 			},
+			"expiration_leeway": {
+				Type: framework.TypeDurationSecond,
+				Description: `Duration in seconds of leeway when validating expiration of a token to account for clock skew. 
+Defaults to 150 (2.5 minutes), minimum of 1 second.`,
```

Comment:
Post beta1, can you add text here (and in the other options you added) indicating that setting to 0 will use the default value?